include/SFML/Audio/InputSoundFile.hpp

```diff
@@ -63,8 +63,11 @@ class SFML_AUDIO_API InputSoundFile : NonCopyable
     ////////////////////////////////////////////////////////////
     /// \brief Open a sound file from the disk for reading
     ///
-    /// The supported audio formats are: WAV (PCM only), OGG/Vorbis, FLAC.
+    /// The supported audio formats are: WAV (PCM only), OGG/Vorbis, FLAC, MP3.
     /// The supported sample sizes for FLAC and WAV are 8, 16, 24 and 32 bit.
+    /// Because of minimp3_ex limitation, for MP3 files with big (>16kb) APEv2 tag,
+    /// it may not be properly removed, tag data will be treated as MP3 data
+    /// and there is a low chance of garbage decoded at the end of file.
```

Comment:
Thanks for this update! I've read again your [GitHub comment](https://github.com/SFML/SFML/issues/1232#issuecomment-641633168) explaining things. Maybe as you suggested, we can still provide a link to the ReadMe:

```cpp
/// See also: https://github.com/lieff/minimp3
```

The ReadMe in https://github.com/lieff/minimp3 states:

> At end of stream just pass rest of the buffer, sync procedure should work even with just 1 frame in stream (except for free format and **garbage at the end can mess things up, so id3v1 and ape tags must be removed first**).

Possibly we should also create an issue for this problem, and address it later so _all_ MP3 files can be read, and we can get rid of this leaky abstraction. It's of course a bit tricky, since the API relies on chunk-by-chunk streaming without lookahead.

Reply:
Since this essentially is a limitation of minimp3, do we need to create an issue on our side?

Reply:
I'm not sure, is it really a limitation of minimp3 in the sense that a better _implementation_ (not API) could address it? I understood that as a consequence of minimp3's streaming/batch processing, it cannot detect the id3v1/ape tag when it's larger than one batch window (16kB).

Also, for the SFML user, minimp3 is an implementation detail. If some MP3 files cannot be played, that still negatively affects user experience. But I'm also OK if we only create an issue when it's becoming a real problem; this pull request should contain enough information.

Reply:
I don't really like to open issues, unless we have a clear plan on how to tackle it. When users open issues, then we at least know that it has a "real world" impact, while this here is just theoretical guessing (unless we notice that it's a more common issue, during testing).

Reply:
That's a good take 👍